### DISCUSSION TITLE
[Enhancement] Allow saving empty tags with pet new -t

### DISCUSSION THREAD

**Discussion by craSH** [knqyf263/pet#269] [Ideas]
I like to use `pet new -t` as my default invocation (via a shell key bind), but sometimes I may not want to specify tags at the moment. It'd be nice if just hitting enter/saving an empty string in the Tags prompt would proceed as though `-t` were not specified, instead of needing to abort entirely at that point if you can't think of any tags to enter.

Thanks for the tool, I'm early on in my usage but I think it's going to be a lasting one!

**Comment by RamiAwar** [comment]
Thanks for the idea @craSH! I like it, we can make this happen. Definitely makes sense UX-wise.

**Comment by RamiAwar** [comment]
Just merged this [c17deaf](https://github.com/knqyf263/pet/commit/c17deafdac174d7f570f1c771fd7ce81a80a8171)! 

Now if you enter an empty value, it will just save the command without any tags.
